tests/test_gameboard.py

```diff
@@ -120,3 +120,55 @@ def test_winner_exists_from_diagonals_on_right_corner(self):
         self.assertEqual(self.board_test_diagonals.winner_exists(), False)
         self.board_test_diagonals.put_chip(7, 'red')
         self.assertEqual(self.board_test_diagonals.winner_exists(), True)
+
+    def test_equality_of_empty_boards(self):
+        board_1 = GameBoard.from_matrix([
```

Comment:
Very minor thing: you could have an `empty()` class method to do this (`GameBoard.empty()`)